### DISCUSSION TITLE
How can I get copilot for free ?

### DISCUSSION THREAD

**Discussion by hemedani** [community/community#53569] [Copilot Conversations]
I have several Open source projects like [lesvim](https://github.com/MiaadTeam/lesvim) and [lesan](https://github.com/MiaadTeam/lesan), but I can't get a free license.
I'm from Iran and it's very hard for me to get a subscription for a paid account

**Comment by duriann** [comment]
Could it be that the number of starts of these open source projects is not enough?

**Reply by hemedani** [reply]
Do you know how many stars are required for that?
